### PR TITLE
Add AnaMaria's into to README.md

### PR DESCRIPTION
**Pull request by amarangunic**
@eculler Hey Elsa adding my info to the readme. See you on wednesday at 4 pm.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,5 +5,5 @@ Welcome to the Earth Analytics Program! You will use this repository to ask ques
 Contributors:
 | Name | GitHub Username | Portfolio URL |
 | ---- | --------------- | ------------- |
-|  |  |  |
+| Ana Maria Marangunic | amarangunic |amarangunic.github.io  |
```
Comment (Lauren-Gleason) [inline README.md:8]: Looks good!

**Review by Lauren-Gleason — APPROVED** [review]
Everything looks good!

**Comment by nquarder** [comment]
Hey @amarangunic - nice work with this so far. 

It looks like the text you added to your portfolio page in the 3rd column isn't linked. You can edit this text using Markdown to link to your portfolio page.

The syntax will look something like `[username.github.io](username.github.io)` 

Here is a link to the Markdown guide that explains the syntax in more detail: https://www.markdownguide.org/basic-syntax/#links

You can try re-editing the README.md file to include this Markdown syntax and then resubmitting the Pull Request. Thanks! -NQ

**Review by eculler — APPROVED** [review]
Looks great, @amarangunic , the link works!

**Comment by amarangunic** [comment]
Thanks Elsa and Nathan for you help!
Excelent!

**Review by tjstogoski — APPROVED** [review]
Looks great! Link works for me.

**Pull request merged**